File: 241023/일 수 구하기/number-of-days-in-month.cpp
#include <iostream>
using namespace std;

int main() {
    int month;
    cin >> month;

    if (month == 2) {
        cout << 28;
    } else if (month <= 7) {
        if (month % 2 == 1) {
            cout << 31;
        } else {
            cout << 30;
        }
    } else {
        if (month % 2 == 0) {
            cout << 31;
        } else {
            cout << 30;
        }
    }
    return 0;
}

File: 241023/일 수 구하기/number-of-days-in-month.js
const fs = require("fs");
let month = Number(fs.readFileSync(0).toString().trim());

if (month === 2) {
    console.log(28);
} else if (month <= 7) {
    if (month % 2 === 1) {
        console.log(31);
    } else {
        console.log(30);
    }
} else {
    if (month % 2 === 0) {
        console.log(31);
    } else {
        console.log(30);
    }
}